info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -283,6 +288,10 @@ private void storeElectionContestCandidateInDatabase(JsonObject candidate,
         Entity.newBuilder(candidateKey)
             .set("name", name)
             .set("partyAffiliation", party + " Party")
+            .set("email", email);
+            .set("phone number",phoneNumber);
+            .set("photoURL", photo);
```

Comment:
You're using photoURL here vs photoUrl when doing candidate.get(). I would keep  it consistent.

Reply:
the get is for the the API and the set is for  the database. If I change the column name in the database to match the API it  will break other existing code but it is an easy fix.